prog: command not found - when testing argcomplete 1.6.0 from PyPI

I'm trying to package argcomplete in the Guix distribution and I get lots of lines like the following lines when testing:

```shell
======================================================================
FAIL: test_unquoted_space (test.TestBash)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/guix-build-python-argcomplete-1.6.0.drv-0/argcomplete-1.6.0/test/__init__.py", line 758, in test_unquoted_space
    self.assertEqual(self.bash.run_command('prog space f\t'), 'foo bar\r\n')
AssertionError: 'bash: prog: command not found\r\n' != 'foo bar\r\n'
- bash: prog: command not found
+ foo bar
```

Am I missing something?

Also, I've had to rename `test/test.py` to `test/__init__.py` for the tests to run at all. Do I need a special test runner? I've tried nose and pytest so far - no luck.

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,2 +1,3 @@
 include argcomplete/bash_completion.d/*
 include *.rst
+include test/*

